docs/Installation/Installation.md

```diff
@@ -327,77 +282,71 @@ yaml-cpp@develop~shared` (note that is a tilde and not a dash in front
+  run `git checkout v6.10.2` to switch to a supported, stable release of
+  Charm++.
+- Choose the `LIBS` target to compile. This is needed so that we can support the
+  more sophisticated load balancers in SpECTRE executables.
+- On a personal machine the correct target architecture is likely
+  `multicore-linux-x86_64`, or `multicore-darwin-x86_64` on macOS. On an HPC
+  system the correct Charm++ target architecture depends on the machine's
+  inter-node communication architecture.
+- Both Charm++ and SpECTRE must be compiled using the same compiler,
   otherwise you will receive undefined reference errors while linking SpECTRE.
   When compiling Charm++ you can specify the compiler using, for example,
   ```
-  ./build charm++ ARCH clang
-  ```
-  When compiling SpECTRE you can specify the compiler to CMake using,
-  for example,
-  ```
-  cmake -D CMAKE_CXX_COMPILER=clang++ \
-        -D CMAKE_C_COMPILER=clang \
-        -D CMAKE_Fortran_COMPILER=gfortran \
-        -D CHARM_ROOT=CHARM_DIR/ARCH_OPTS SPECTRE_ROOT
+  ./build LIBS ARCH clang
   ```
-* Inside the SpECTRE build directory, use `make list` to see all available
-  targets. This list can be refreshed by running CMake again.
+
+## Building SpECTRE {#building-spectre}
+
+Once you have set up your development environment you can compile SpECTRE.
+Follow these steps:
+
+1. Create a build directory where you would like to compile SpECTRE. In the
+   Docker container you could create, e.g., `/work/spectre-build`. It can be
+   useful to add a timestamp or descriptive label to the name of the build
+   directory since you may create more later, e.g., `build_YYYY-MM-DD` or
+   `build-clang-Debug`. Then, `cd` into the build directory.
+2. Determine the location of your Charm++ installation. In the Docker container
+   it is `/work/charm_6_10_2/multicore-linux-x86_64-gcc` for GCC builds and
+   `/work/charm_6_10_2/multicore-linux-x86_64-clang` for clang builds. For Spack
+   installations you can determine it with
+   `spack location --install-dir charmpp`. We refer to the install directory as
+   `CHARM_ROOT` below.
+3. In your new SpECTRE build directory, configure the build with CMake:
+   ```
+   cmake -D CHARM_ROOT=$CHARM_ROOT SPECTRE_ROOT
+   ```
+   Add options to the `cmake` command to to configure the build, select
+   compilers, etc. For instance, to build with clang you may run:
+   ```
+   cmake -D CMAKE_CXX_COMPILER=clang++ \
+         -D CMAKE_C_COMPILER=clang \
+         -D CMAKE_Fortran_COMPILER=gfortran \
+         -D CHARM_ROOT=$CHARM_ROOT \
+         SPECTRE_ROOT
+   ```
+   See \ref common_cmake_flags for documentation on possible configuration
+   options.
+4. When cmake configuration is done, you are ready to build target executables.
+   Compile unit tests with `make unit-tests -jN` where `N` is the number of
+   cores to build on in parallel (e.g. `-j4`).
+
+   - You can see the list of available targets by running `make list`.
```

Comment:
"or by using tab completion. Note that the Ninja generator allows you to compile individual source files too."

Reply:
Reworded this a bit, so pushed a fixup